sheetsync/sheetsync/main.py

```diff
@@ -413,7 +417,14 @@ class PlaylistsSync(SheetSync):
 			event_id: The id of the streamlog event to sync
 	""",
 )
-def main(dbconnect, sync_configs, metrics_port=8005, backdoor_port=0, media_dir="."):
+@argh.arg('--timezone', help="Local timezone for determining shift times")
```

Comment:
Could we make the timezone part of the `shifts` json instead? It removes the possibility of giving one without the other, when they only make sense together. It can be another top-level key.
It would also mean one less variable to need to plumb through everything.

Reply:
Good idea.